README.md

```diff
@@ -25,3 +26,22 @@ while  not board.is_game_over() and  not board.can_claim_draw():
 
 engine.quit()
 ```
+
+## Training
+
+To run the training process, first one should install the python project on `venv`:
+```bash
+pip install poetry
+
+mkdir ~/.venvs; cd ~/.venvs
+python -m venv cattus

```

Comment:
I think we don't need to explain all this `venv` stuff - it's a general Python environment, relevant to all Python projects. Most projects just tell you to `pip install <project_name>` -- if you want to create a different environment for it, you can do it.